info/v1/container.go

```diff
@@ -874,6 +874,19 @@ type ProcessStats struct {
 	Ulimits []UlimitSpec `json:"ulimits,omitempty"`
 }
 
+type MemoryBandwidthMonitoringStats struct {
+	// The 'mbm_total_bytes'.
+	TotalBytes uint64 `json:"mbm_total_bytes,omitempty"`
+
+	// The 'mbm_local_bytes'.
+	LocalBytes uint64 `json:"mbm_local_bytes,omitempty"`
```

Comment:
Have you removed LLC stats completely? Why?

Reply:
I thought that WCA doesn't collect this... I'll add this in a CMT stats.

Reply:
We should expose it if we expose RDT based stats.